code/script.js

```diff
@@ -1,51 +1,199 @@
 // Variables that point to selected DOM elements
-const chat = document.getElementById('chat');
+const chat = document.getElementById('chat')
+const inputWrapper = document.getElementById("input-wrapper")
+const nameInput = document.getElementById("name-input")
+const button = document.getElementById("button")
 
-// If you need any global variables that you can use across different functions, declare them here:
+//global variables 
+let questionNumber = 1
 
+//Sound effects from Pixabay
+const soundEffectBot = () => {
+  const sound = new Audio('short-woosh-109592.mp3');
+  sound.play()
+}
+const soundEffectUser = () => {
+  const sound = new Audio('short-success-sound-glockenspiel-treasure-video-game-6346.mp3');
+  sound.play()
+}
+
+const soundEffectFinal = () => {
+  const sound = new Audio('sport-rock-logo1-13776.mp3');
+  sound.play()
+}
+
```

Comment:
Nice idea for the sound effects to be functions and declaring before using them. Perhaps if they were simply variables that you then use where you need them using the `.play()` method could have also worked.